File: WebUI/ClientApp/src/app/core/entity/customer/customer.ts
import { Account } from "../account/account";

export class Customer {
  public id: number;
  public firstName: string;
  public lastName: string;
  public username: string;
  public phoneNumber: string;
  public birthday: Date;
  public createAt: Date;
  public modifyAt: Date;
  public rememberMe: boolean;
  public tearmsAndContidions: boolean;

  public accountId: number;
  public account: Account = new Account();

  constructor() { }
}

File: Domain/Entities/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Customer
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Username { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime? Birthday { get; set; }
        public DateTime CreateAt { get; set; }
        public DateTime ModifyAt { get; set; }

        public int AccountId { get; set; }
        public Account Account { get; set; }
    }
}
